src/tests/Makefile.manual

```diff
@@ -14,6 +14,7 @@ testdrive.F90:
 all test clean::
 	$(MAKE) -f Makefile.manual --directory=ascii $@
 	$(MAKE) -f Makefile.manual --directory=bitsets $@
+	$(MAKE) -f Makefile.manual --directory=hash_functions_perf $@
 	$(MAKE) -f Makefile.manual --directory=hash_functions $@
```

Comment:
I believe that this will automatically try to run the tests for the hash validation codes, and currently fail if the library is compiled with OneAPI. I believe we have three choices:
1. Fix the C conditional compilation code for the nmhashes so it never tries to use `___builtin_rotateleft32`; or
2. Add constraints in the Makefile so that it will only automatically run the hash validation codes if the GCC compiler suite is being used (I am not an expert on this aspect of Makefiles); or
3. Only allow the hash validation codes to be run manually.